_visualisation/core/summarisation.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Summarisation of Interests
+short_title: Summarisation
+position: 1.11
+---
+
+You may wish to summarise the interests of more than one Party as a single figure. For instance, to summarise the ownership interests of multiple Persons, or to give the ultimate result of a Chain of control interests.
+{: .lead}
+
+Summarised figures might appear in the captions to your diagram, in its supporting text, or as part of the diagram itself through [Optional Features](/visualisation/optional) such as [Indirect Arrows](/visualisation/optional/annotation) or [Charts](/visualisation/optional/charts).
+
+* Sections
+{:toc}
+
+Summarisation results in a quantified interest, that is, a percentage figure, even if the interests that were summarised were [obscured](/visualisation/core/unknowns) or not quantified. However, the summarised figure may contain [uncertainty](/visualisation/core/unknowns) (that is, it may be a range, e.g. 10-20%).
+
+These summarised figures must obviously be mathematically accurate. This can be tricky to calculate when interests are [obscured](/visualisation/core/unknowns). Take care to follow the arithmetic rules below to ensure summarised figures are correct.
+
+
+## Overview
+
+In general, interests are summarised in the following way:
+
+* To **combine** interests of multiple Parties in a single Entity, use **addition**.
+* To give the **compounded** interest of one Party, via intermediaries, in another, use **multiplication**.
+
+Summarisation must focus on interests of a single nature, i.e. either ownership or control. In the below, we write just "interest" for brevity; this should be taken to mean "interest of the selected nature".
+
+
+## Combined Interests (Addition)
+
+![Combined Interests in a BOVS Diagram](/visualisation/diagrams/bovs-core-summarisation-combined.png){: style="width: 87.5%" }
+
+You may summarise the interests of multiple Parties in a given Entity by adding together each of their direct interests in that Entity. The summarised figure is referred to as their **combined interest** in the given Entity.
+
+* Any Party that does not have a direct interest in the selected Entity is treated as a zero in the addition.
+* See below for how to handle addition in cases where interests are obscured.
+
+
```

Comment:
I think we may benefit from a box-out on 'why interests may sum to more than 100%'. Suggested text below:

-----
**Why interests may sum to more than 100%** 

It many cases it may not make logical sense to have a particular kind of interest sum to more than 100%. However, diagrams should be faithful to the data they are drawn from, and that data may indicate interests that sum to more than 100%. 

This may be because:

* The data is taken from different points in time;
* The data contains errors;
* The data contains ranges rather than absolute values

In each of these cases, seeing that the data may indicate interests of more than 100% is informative, highlighting the need for further checks on the data, or for care to be taken in data interpretation. 

**If a diagram rounds down interests to 100% because of a clear use case that requires this, a clear note should be included to this effect.**

----

_I think this last caveat may be needed, to allow for certain use cases to set a ceiling of 100% - but not sure if this should be phrased as a rule, or just a note_.

Reply:
There are two ways of reaching more than 100% as far as I can tell:

1. Source data unambiguously sums to more than 100% – very indicative of a big issue with the data
2. Source data has uncertainty, so the uncertain part goes over 100%, but the certain part does not

I don't think we should allow for diagrams/data to be reported with a ceiling, as in case (1) this directly misrepresents the data; and in (2) it misrepresents the level of uncertainty.

As an example of case (2), if ownership is 40-75% + 40-75% = 80-150%, we have a total unknown portion of 70%. To show this as 80-100% makes it seem like there is far less uncertainty than there in fact is.

What do you think?

Reply:
@Convincible - How about changing Tim's caveat wording to: "It's not advisable to round summed interests down to 100%. If such manipulation is unavoidable for a particular use case, provide a clear explanation of the process and rationale."